lnurl-withdraw.md

```diff
@@ -77,8 +77,8 @@ Eg:
 	https://LNserviceURL
 	?tag=withdrawRequest
 	&k1=String
-	&minWithdrawable=MilliSatoshi
-	&maxWithdrawable=MilliSatoshi
+	&minWithdrawable=Integer
+	&maxWithdrawable=Integer
```

Comment:
I personally find specifying units much easier to understand. What is the reason for this change?

Reply:
I agree. Especially since the lnurl-channel spec uses whole sats instead of msats.

Reply:
I just find it clearer this way because "MilliSatoshi" is not a valid JSON type. But I can change this back.